docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,29 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+By default, Amazon DynamoDB is available for all AWS users. The Scalar DL/Scalar DB schema loader will set the required configurations while creating the schema. You do not need to set up anything manually.
+
+### Steps
+
```

Comment:
I think you can add one more optional step to change the read and write capacity of DynamoDB tables.

Reply:
I will add the specified step.
Thank you for the suggestion.